search/intermediate_result.cpp

```diff
@@ -281,10 +281,16 @@ void FillDetails(FeatureType & ft, Result::Details & details)
     }
   }
 
-  if (strings::to_uint(ft.GetMetadata(feature::Metadata::FMD_STARS), details.m_stars))
+  details.m_isHotel = feature::IsHotel(ft);
```

Comment:
IsHotel function is not needed, as I mentioned in another comment. It would be better to use its body here instead.